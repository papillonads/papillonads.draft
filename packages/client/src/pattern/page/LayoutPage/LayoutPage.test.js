import React from 'react'
import layoutPageDefault from './LayoutPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<LayoutPage />', () => {
  describe('Snaphot', () => {
    test('must match layoutPageDefault', () => {
      expect(global.renderToJSON(layoutPageDefault)).toMatchSnapshot()
    })
  })
})
