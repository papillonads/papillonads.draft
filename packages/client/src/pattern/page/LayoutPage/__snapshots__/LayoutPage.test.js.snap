// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutPage /> Snaphot must match layoutPageDefault 1`] = `<div />`;
